app/models/case/base.rb

```diff
@@ -572,7 +572,7 @@ def responded_in_time_for_stats_purposes?
   end
 
   def already_late?
-    Date.today > external_deadline
+    Date.current > external_deadline
   end
```

Comment:
Nice.  I didn't know the difference between `Date.today` and `Date.current` until recently.